sdk/metricsadvisor/ai-metrics-advisor/package.json

```
@@ -137,6 +137,18 @@
       "metricQueries.js",
       "quickstart.js",
       "seriesData.js"
-    ]
+    ],
+    "productName": "Azure Metrics Advisor",
+    "productSlugs": [
+      "azure",
+      "azure-cognitive-services",
+      "azure-metrics-advisor"
+    ],
+    "requiredResources": {
+      "Azure Cognitive Services account": "https://docs.microsoft.com/azure/cognitive-services/cognitive-services-apis-create-account"
+    },
+    "skipFolder": false,
+    "disableDocsMs": false,
+    "apiRefLink": "https://docs.microsoft.com/javascript/api/@azure/ai-metrics-advisor/?view=azure-node-preview"
```

```suggestion
    "apiRefLink": "https://docs.microsoft.com/javascript/api/@azure/ai-metrics-advisor/"
```
The link should work now (redirecting to preview) and in the future